WebContent/member page

```diff
@@ -0,0 +1,24 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Member Page</title>
+</head>
+<body>
+
+<h1>Welcome to your home page</h1>
+
+<div>
+<form>
```

Comment:
instead of putting an `onclick` onto a `button`, why not use the `action` attribute of the `<form>` tag, and change this button to a `submit` button?

see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form for more info